dev/archery/archery/crossbow/cli.py

```diff
@@ -310,9 +308,6 @@ def asset_callback(task_name, task, asset):
               help='Crossbow repository on github to use')
 @click.option('--fetch/--no-fetch', default=True,
               help='Fetch references (branches and tags) from the remote')
-@click.option('--github-token', envvar='ARROW_GITHUB_API_TOKEN',
```

Comment:
don't we have to remove `github_token` from the function signature too?
```
def report_pr(obj, arrow_remote, crossbow, fetch, github_token, job_name,
```

Reply:
Good catch!
I'll fix it.

Reply:
Fixed.